docs/langref/src/types.md

```diff
@@ -9,7 +9,7 @@ All properties in elements have a type. The following types are supported:
 | `bool`               | boolean whose value can be either `true` or `false`.                                                                                                                                                                                                                                                                                             |
 | `string`             | UTF-8 encoded, reference counted string.                                                                                                                                                                                                                                                                                                         |
 | `color`              | RGB color with an alpha channel, with 8 bit precision for each channel. CSS color names as well as the hexadecimal color encodings are supported, such as `#RRGGBBAA` or `#RGB`.                                                                                                                                                                 |
-| `brush`              | A brush is a special type that can be either initialized from a color or a gradient specification. See the [Colors Section](#colors) for more information.                                                                                                                                                                                       |
+| `brush`              | A brush is a special type that can be either initialized from a color or a gradient specification. See the [Colors and Brushes Section](#colors-and-brushes) for more information.                                                                                                                                                                                       |
```

Comment:
Great find!

This doesn't seem entirely correct to me though, I think the link needs to also include the file name, since the target anchor is not in types.md:

```suggestion
| `brush`              | A brush is a special type that can be either initialized from a color or a gradient specification. See the [Colors and Brushes Section](brushes.md#colors-and-brushes) for more information.                                                                                                                                                                                       |
```

Reply:
That's possible. I followed the style that was already there. In the end, these multiple files seem to be glued together into one page, both in Rust and c++ docs so it does work.

I don't have a full setup to generate the docs on my machine though. Gotta do it at some point finally :)

Reply:
Ahh, there's a mismatch - I see it now. In #2151 @hunger began moving the language reference (including this section) from a monolithic file (`langref.md`) into multiple separate files, that are compiled with `mdbook`. The issue that you saw exists in the original `langref.md` (where your change is exactly correct!) as well as in the fork, where the types are split out into `docs/langref/src/types.md` (the file that you change edits).

Unfortunately your change crossed with @hunger's :)

So my above suggestion is correct for the new mdbook build. If you want to try it out, btw, it's fairly straightforward:

* `cargo install mdbook`
* Then go into `docs/langref`
* Run `mdbook build`
* open `book/html/index.html`

For the monolithic `langref.md` in `docs/langref.md` your change is correct (but the path is wrong).

Reply:
Thanks for the tips.